RELEASE.md

```diff
@@ -4,21 +4,24 @@
 
 To release a version _“v0.W.Z”_ of Kuadrant Operator in GitHub and Quay.io, follow these steps:
 
-1. Kuadrant dependencies need to be released first:
+1. Verify correct versions for the Kuadrant release and its dependencies are listed in the `release.toml` file.
+   The verification should be done as part of the community call prior to cutting the release.
+   Update versions if required, release of dependency maybe required, see step 2.
+   This file is located in the root of the kuadrant operator.
+
+2. (Optional) Release Kuadrant dependencies:
    * [Authorino Operator](https://github.com/Kuadrant/authorino-operator/blob/main/RELEASE.md).
    * [Limitador Operator](https://github.com/Kuadrant/limitador-operator/blob/main/RELEASE.md).
    * [DNS Operator](https://github.com/Kuadrant/dns-operator/blob/main/docs/RELEASE.md).
    * [WASM Shim](https://github.com/Kuadrant/wasm-shim/).
    * [Console Plugin](https://github.com/Kuadrant/kuadrant-console-plugin).
 
-2. Run the GHA [Release operator](https://github.com/Kuadrant/kuadrant-operator/actions/workflows/release.yaml); make
-   sure to fill all the fields:
+   Once dependencies have being released jump to step 1.
 
+3. Run the GHA [Release operator](https://github.com/Kuadrant/kuadrant-operator/actions/workflows/release.yaml); make
+   sure to fill all the fields:
     * Branch containing the release workflow file – default: `main`
```

Comment:
are there obvious times where these will be different values?

Reply:
At this stage, I am not sure. This is still the draft and that was what the existing was doing so didn't change it. I will look at this again.